docs/guide/troubleshooting.md

```diff
@@ -6,19 +6,19 @@ This document covers some common Vue CLI issues and how to resolve them. You sho
 
 If you install `@vue/cli-service` as `root` user or with `sudo`, there might be issues when running package `postinstall` scripts.
 
-This is a security feature of npm. You should always avoid running npm with root privileges because install scripts can be unintentionally malicious.
+This is a security feature of npm. You should always avoid running npm with root privileges because installing scripts can be unintentionally malicious.
```

Comment:
```suggestion
This is a security feature of npm. You should always avoid running npm with root privileges because installation scripts can be unintentionally malicious.
```